src/parsing.jl

```diff
@@ -1,5 +1,5 @@
 "A wrapper for one value in a PostgreSQL result."
-struct PQValue{OID}
+struct PQValue{OID, binary_format}
```

Comment:
If we put it in a type parameter here, we should put it in a type parameter in `Result`